jenkins/job-configs/kubernetes-jenkins-pull/charts-pull.yaml

```diff
@@ -0,0 +1,138 @@
+            branches:
+                - 'origin/pr/${{ghprbPullId}}/merge'
+            basedir: '{git-basedir}'
+            browser: githubweb
+            browser-url: 'http://github.com/{repo-name}'
+            timeout: 20
+            clean:
+                after: true
+            wipe-workspace: false
+            skip-tag: true
+    triggers:
+        - github-pull-request:
+            # This is the Jenkins GHPRB plugin ID, not the actual github token.
+            auth-id: 'f8e31bc1-9abb-460a-a2ca-9c4aae3ca4e8'
+            only-trigger-phrase: '{only-trigger-phrase}'
+            trigger-phrase: '(?is).*@k8s-bot\s+(e2e )?test\s+this.*'
+            cron: 'H/2 * * * *'
+            status-context: Jenkins Charts e2e
+            status-url: 'https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/${{ghprbPullId}}/${{JOB_NAME}}/${{BUILD_NUMBER}}/'
+            status-add-test-results: '{status-add-test-results}'
+            # This should roughly line up with charts-maintainers.
+            # To go to :
+            #   https://github.com/orgs/kubernetes/teams/charts-maintainers
+            admin-list:
+                - k8s-merge-robot
+                - jackgr
+                - adamreese
+                - bmelville
+                - sgoings
+                - sparkprime
+                - vaikas-google
+                - viglesiasce
+                - etune
+                - prydonious
+                - runseb
+                - technosophos
+                - michelleN
+            # This should roughly line up with charts-collaborators.
+            # To go to :
+            #   https://github.com/orgs/kubernetes/teams/charts-collaborators
+            white-list:
+                - runseb
+                - vaikas-google
+                - jackgr
+            org-list:
+                - kubernetes
+            white-list-target-branches:
+                - master
+            success-comment: '{success-comment}'
+            failure-comment: '{failure-comment}'
+            error-comment: '{error-comment}'
+    wrappers:
+        - e2e-credentials-binding
+        - inject:
+            properties-content: |
+                GOROOT=/usr/local/go
+                GOPATH=$WORKSPACE/go
+                PATH=$PATH:$GOROOT/bin:$WORKSPACE/go/bin
+        - workspace-cleanup:
+            dirmatch: true
+            external-deletion-command: 'sudo rm -rf %s'
+        - timeout:
+            timeout: 90
+            fail: true
+        - ansicolor:
+            colormap: xterm
+    builders:
+        - activate-gce-service-account
+        - ensure-upload-to-gcs-script:
+            git-basedir: '{git-basedir}'
+        - shell: JENKINS_BUILD_STARTED=true "${{WORKSPACE}}/_tmp/upload-to-gcs.sh"
+        - shell: |
+              if [[ -n '{git-basedir}' ]]; then cd '{git-basedir}'; fi
+              ./test/e2e.sh
+    publishers:
+        - xunit:
```

Comment:
are your tests producing any junit? if not, you can remove this section entirely.